zipline/assets/asset_writer.py

```diff
@@ -432,6 +445,8 @@ def write(self,
                   A short description of this root symbol.
               exchange : str
                   The exchange where this root symbol is traded.
+        supplementary_mappings : pd.DataFrame, optional
+            Additional mappings from values of abitrary type to assets.
```

Comment:
It looked like the foreign key is to the equities table.  Do we want to support lookup to any asset class or just equities, and if it's just equities, should we name these various objects more specifically, e.g. `equity_supplementary_mappings`?